Date: Mar 18, 2021
From: Pavithra <Pavithra_Pandian@comcast.com>
Source: COMCAST
Subject: Packager has been disabled by mistake as part
of RDK-29622 US by Wifi team. Enabling it back
Signed-off-by: Pavithra <Pavithra_Pandian@comcast.com>
Index: opkg-0.4.2/libopkg/pkg_extract.c
===================================================================
--- opkg-0.4.2.orig/libopkg/pkg_extract.c
+++ opkg-0.4.2/libopkg/pkg_extract.c
@@ -96,7 +96,7 @@ int pkg_extract_rdm_data_files_to_dir(ch
         return -1;
     }
 
-    r = ar_extract_all(ar, dir);
+    r = ar_extract_all(ar, dir, NULL);
     if (r < 0)
         opkg_msg(ERROR, "Failed to extract data files from package '%s'.\n",
                  filename);
Index: opkg-0.4.2/libopkg/opkg_archive.c
===================================================================
--- opkg-0.4.2.orig/libopkg/opkg_archive.c
+++ opkg-0.4.2/libopkg/opkg_archive.c
@@ -952,7 +952,7 @@ int ar_extract_rdm_all(char *filename, c
     if (opkg_config->ignore_uid)
         extract_flags &= ~ARCHIVE_EXTRACT_OWNER;
 
-    r = extract_all(ar, dir, extract_flags);
+    r = extract_all(ar, dir, extract_flags, NULL);
 
  err_cleanup:
     archive_read_free(ar);
